src/components/dev-hub/blog-post-title-area.js

```diff
@@ -46,13 +46,15 @@ const BlogPostTitleArea = ({
     tags,
     title,
     updatedDate,
+    maintainSquareAspectRatio,
 }) => {
     const BlogTitle = H2.withComponent('h1');
     return (
         <HeroBanner
             background={articleImage}
             breadcrumb={breadcrumb}
             data-test="hero-banner"
+            maintainSquareAspectRatio={maintainSquareAspectRatio}
```

Comment:
Hero page uses maintainSquareRatio parameter, which is not being passed from blog post title area, so it takes the default true parameter. I need to pass this parameter as false to remove extra spacing , which is coming up in case of mobile display

Reply:
Absolutely makes sense, I agree